work_dir_tools.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+#variables
+MODE=$1
+SDL_BACKUP=${TMPDIR-/tmp}"/sdL_backup"
+SDL=${2%/}
+
+#functions
+usage() {
+    echo "Usage: env.sh [OPTION] [PATH]"
+    echo "Path is a link to SDL bin folder"
+    echo "Mode can be the following:"
+    echo "  -b, --backup - Back up SDL configuration"
+    echo "  -c, --clean - Remove temporary SDL working files"
+    echo "  -r, --restore - Restore SDL configuration from backup"
+    echo "  -h, --help - Display this help"
+}
+
+backup() {
+	echo "Backup environment"
+	rm -r -f $SDL_BACKUP
+	mkdir $SDL_BACKUP
+	if [ -d $SDL_BACKUP ]; then
+		cp $SDL"/hmi_capabilities.json" $SDL_BACKUP
+		cp $SDL"/sdl_preloaded_pt.json" $SDL_BACKUP
+		cp $SDL"/smartDeviceLink.ini" $SDL_BACKUP
+		echo "Done"
+	else
+		echo "Folder '"$SDL_BACKUP"' doesn't exists"
```

Comment:
@dboltovskyi it is unclear message for user, because you just create in script. It should not exist during script running. 
"Unable to create $SDL_BACKU folder " is better